CoreScatterPlotView/src/au/gov/asd/tac/constellation/views/scatterplot/ScatterOptionsPane.java

```diff
@@ -343,7 +344,7 @@ protected void enableOptions() {
     /**
      * Write the given ScatterPlotState to the active graph.
      */
-    @PluginInfo(tags = {"LOW LEVEL"})
+    @PluginInfo(pluginType = PluginType.UPDATE, tags = {"LOW LEVEL"})
     private static class ScatterPlotStateWriter extends SimpleEditPlugin {
```

Comment:
Duplication of a scatter plot state updater? Maybe drag out into its own class so it can be used more widely?

Reply:
Haha I'll admit I wasn't paying close enough attention to pick that up. I'll make that change